arviz/plots/separationplot.py

```diff
@@ -0,0 +1,112 @@
+"""Separation plot for discrete outcome models."""
+from ..rcparams import rcParams
+from .plot_utils import get_plotting_function
+
+
+def plot_separation(
+    idata=None,
+    y=None,
+    y_hat=None,
+    y_hat_line=False,
+    expected_events=False,
+    figsize=None,
+    textsize=None,
+    color=None,
```

Comment:
should we set `color="C0"` here and then convert to hex before passing it to the backend specific function? This is more a philosophical question about whether we want this behaviour than anything else.

Reply:
Yeah, I think we want this everywhere, not only here.